ci/build_wince.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
+mkdir /var/lib/apt/lists/partial
+apt-get update
+apt-get install ssh git
```

Comment:
This should probably be added here instead : https://github.com/navit-gps/Dockerfiles/blob/master/wince/Dockerfile#L3

Reply:
Agreed :+1: